doc/construct_template.rst

```diff
@@ -46,11 +49,12 @@ wavelength ranges, and master wavecalib files into a final template.
 
 The input looks like the following:
 
-    templates.build_template(wfiles, slits, wv_cuts, binspec, outroot,
-     ifiles=ifiles, det_cut=det_cut, chk=True,
-     normalize=False, lowredux=False,
-     subtract_conti=True, overwrite=overwrite,
-     shift_wave=True)
+.. code-block:: python
+
+    from pypeit.core.wavecal import templates
+
+    templates.build_template(wfiles, slits, wv_cuts, binspec, outroot, ifiles=ifiles, det_cut=det_cut, chk=True,
+                             normalize=False, lowredux=False, subtract_conti=True, overwrite=overwrite, shift_wave=True)
 
-This produces a file called 'outroot' that contains the template. It also
+This produces a file called `outroot` that contains the template. It also
```

Comment:
I mix this up, as well, but I think if you want  outroot to appear like `outroot`, you need to use two quote characters; i.e.,
```
``outroot``
```

It's a difference in Markdown and restructuredText syntax.  As it is, I think this only puts outroot in italics.

Reply:
Thanks! Done.